third_party/ruby/build-ruby.bzl

```diff
@@ -76,7 +76,7 @@ run_cmd() {{
     fi
 }}
 
-run_cmd sed -i.bak -e 's@^COMMONOBJS\( *\)=@COMMONOBJS\\1= {extra_srcs_object_files}@' common.mk
+run_cmd sed -i.bak -e 's@^COMMONOBJS\\( *\\)=@COMMONOBJS\\1= {extra_srcs_object_files}@' common.mk
```

Comment:
Do you know why we're so particular about keeping the exact number of spaces here?  It seems like we could drop all but one with no loss of functionality...unless there's something in Ruby's build system that depends on potential multiple spaces here?

Reply:
I believe we're keeping the spaces so that we match the correct part of the makefile -- matching the space was the easiest way to ensure that we rewrite the right declaration. Maybe a better option would be to rewrite based on the line number, as it's patching something that's not generated by autotools?

Reply:
Maybe `^COMMONOBJS[^=]*=` would be better here? Experimenting on `common.mk` directly, that does seem to match exactly one line.